src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  color: string;
+
+  sideA: number;
+
+  sideB: number;
+
+  sideC: number;
+
+  constructor(color: string, sideA: number, sideB: number, sideC: number) {
```

Comment:
```suggestion
  constructor(public color: string, public sideA: number, public sideB: number, public sideC: number) {
```

Reply:
After I register the public state, for some reason the error Duplicate identifier 'color'.ts(2300) pops up, why this happens?